src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ILLink.targets

```diff
@@ -39,6 +39,10 @@ This file contains the .NET 5-specific targets to customize ILLink
           Value="$(_ProguardProjectConfiguration)"
       />
       <_TrimmerCustomData Include="SystemIOHashingAssemblyPath" Value="$(_SystemIOHashingAssemblyPath)" />
+      <_TrimmerCustomSteps
+        Include="$(_AndroidLinkerCustomStepAssembly)"
+        BeforeStep="MarkStep"
+        Type="Microsoft.Android.Sdk.ILLink.WarnOnPreserveAttribute" />
```

Comment:
Oh, maybe you forgot to commit 1 file.

If this is for .NET 11, I think we could also put `error:true` here:
https://github.com/dotnet/android/blob/68f1238fafe90c32a93619b635b16e90582e0a23/src/Mono.Android/Android.Runtime/PreserveAttribute.cs#L35

It has been marked `[Obsolete]` a long time.

Reply:
Ah, forgot to re-add the file after renaming it.